dags/bq_events_to_amplitude.py

```diff
@@ -21,15 +21,30 @@
         dag_name,
         default_args=default_args) as dag:
 
-    task_id = 'fenix_amplitude_export'
+    fenix_task_id = 'fenix_amplitude_export'
     SubDagOperator(
         subdag=export_to_amplitude(
-            dag_name=task_id,
+            dag_name=fenix_task_id,
             parent_dag_name=dag_name,
             default_args=default_args,
             dataset='telemetry',
             table_or_view='fenix_events_v1',
             s3_prefix='fenix',
         ),
-        task_id=task_id
+        task_id=fenix_task_id
+    )
+
+    fennec_ios_task_id = 'fennec_ios_amplitude_export'
+    fennec_ios_args = default_args.copy()
```

Comment:
This could be done in one line with `.copy().update(..)`

Reply:
Unfortunately `update()` does not return a dictionary instance (just updates it in place).